components/03-organisms/grid/_grid.css

```diff
@@ -1,6 +1,7 @@
 @define-mixin grid {
   display: flex;
   flex-wrap: wrap;
+  margin-bottom: $space-double;
```

Comment:
This is pretty prescriptive. On my last project, it was actually cleaner to do all margins on top.

Reply:
I prefer bottom, but I think what's most important is that it's consistent across the entire project. (always top, or always bottom).

I've based my practice off of this (from IBM, I think?) Specifically number 3: https://docs.google.com/document/d/1YP__VBq0pzyJwqnt1cC8Mr7mYBE7FjNMEe7sAwOMSV4/mobilebasic

Reply:
Let's talk about this though. I'm going to try something, and you can let me know if you like it

Reply:
I don't have a dog in the fight really except that I don't want the base level styles to define it. I always have done bottom as well, but for this project I was just on it made sense within the content layout for them to always be top. Basically, this should be agnostic imo.